develop-docs/sdk/data-model/event-payloads/contexts.mdx

```diff
@@ -814,6 +814,55 @@ Page context contains information about the page that the event occurred on.
 }
 ```
 
+## Outgoing Request Context
+
+Outgoing request context contains information about the outgoing request associated with the event.
+It should only be set for events that are associated with an outgoing request - for example, a `failed to fetch` error on a web page.
+
+`url`
+
+: **Required.** The URL of the outgoing request.
+
+- Example: `https://sentry.io/api/0/projects/sentry/sentry/issues/?page=1`
+
+`method`
+
+: **Required.** The HTTP method of the outgoing request.
+
+- Example: `GET`
+
+`headers`
+
+: **Optional.** The headers of the outgoing request.
+
+- Example: `{ "Content-Type": "application/json" }`
+
+`data`
+
+: **Optional.** The data of the outgoing request.
+
+- Example: `{ "name": "John Doe", "email": "john.doe@example.com" }`
+
+### Example Outgoing Request Context
+
+```json
+{
+  "contexts": {
+    "outgoing_request": {
+      "url": "https://sentry.io/api/0/projects/sentry/sentry/issues/?page=1",
```

Comment:
We do not store query parameters this way. https://develop.sentry.dev/sdk/expected-features/data-handling/#spans

Reply:
Ah, I based this on https://develop.sentry.dev/sdk/data-model/event-payloads/request/#attributes:

> `url` Optional. The URL of the request if available. The query string can be
declared either as part of the url, or separately in query_string.

We can split it out for sure!